EIPS/eip-681.md

```diff
@@ -0,0 +1,78 @@
+## Preamble
+
+    EIP: 681
+    Title: URL Format for Transaction Requests
+    Author: Daniel A. Nagy <daniel@ethereum.org>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Replaces: 67
```

Comment:
This comment from @nicksavers / @pirapira then also applies here: https://github.com/ethereum/EIPs/pull/831#discussion_r161769346